server/controllers/event.js

```diff
@@ -1,25 +1,20 @@
 const Event = require("../models/Event");
-
+const asyncHandler = require("express-async-handler");
 // @route POST /event
-// @desc create an entry for the Event in the database
+// @desc create an entry for the Event
 // @access Private
 
-// Need the data in json foramt and date string should end with z
-// if not UTC time will be used as offset 
-const event = async (req, res) => {
-    const date = new Date(req.body.eventDate);
-    const eventRecord = new Event({
+// We need to send the eventDate with suffix "z" meaning zero time zone.
```

Comment:
Is this comment needed? Also, is there a way we can do this without having to suffix dates with letters?